### ISSUE TITLE
Bulb not showing in Entities/Integrations

### ISSUE THREAD

**Issue by i3lade4life** [fishbigger/HomeAssistant-Tapo-P100-Control#10]
Absolute HA newbie here. I (think I) have done everything correctly to set up 1 of my tapo bulbs. The github file is in the custom_components folder and inserted the lines(including the bulb's IP, TAPO username & password) in to the config.yaml using file editor. The "check configuration" does not come back with any errors before I reset. But when I look on entities there is still no sign of any bulbs. 
I know this probably isn't enough info to go on, but any help would be appreciated.

**Comment by daseijo** [comment]
I have the same problem, I tried to write the lines in the config.yaml in diferents ways and nothing happened. 
Can someone help us?
Thanks

**Comment by fishbigger** [comment]
Is there anything in the HA logs, Errors, Warnings etc.

**Comment by daseijo** [comment]
No. Nothing all is normal. After restart the HA, What should I see? What should I do? 
I check the integrations, and the devices and I don't see anything different. I am sure there is something obvious that i dont know. I tried to find it in hacs also, but is not there. I am really disappointed. The only thing checking the files is that the file called "init" is empty. I was modifying the configuration.yaml, for example, I put "-platform" instead "platform" and I use "switch" instead "light" all my attempts were unsuccessful. 
Thank you for the quick response.

**Comment by fishbigger** [comment]
I can't remember what you see I'll check my install later and see what might be wrong :)

**Comment by daseijo** [comment]
Thank you so much. I am sure is my fault, more people have used your code and it worked.

**Comment by etmipamidzi** [comment]
i have the same issue. Anything like in readme done but nothing happened :(

**Comment by marcotrinelli** [comment]
Same issue. I have the same error above

**Comment by Graeme-SStrong** [comment]
Going by the error in your log, I would say there is a problem with your username or password.

**Comment by diskxo** [comment]
Same problem.

**Comment by diskxo** [comment]
Solved!
I did this schema:

- Delete custom components folder with the configuration.yaml
- restart
- put folder on custom_components;
- restart;
- put code on configuration.yaml file
- restart
- install device on Tapo app;
- restart
- force a password change in Tapo app (i had the old password format, so my password didn’t match the new - -- format);
- edit configuration-yaml
- restart

**Comment by rsg117** [comment]
Solved! -> @diskxo 
thx diskxo
